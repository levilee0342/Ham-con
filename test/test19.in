-13945
-23906
-29625
1008
1205
-1512
-6650
-7064
-4240
-27202
-305
-9958
