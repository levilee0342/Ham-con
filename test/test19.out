1205
-29625